src/App.js

```diff
@@ -1,15 +1,13 @@
 import React from 'react';
-import {BrowserRouter as Router, Routes, Route, Link} from 'react-router-dom';
+import {BrowserRouter as Router, Routes, Route} from 'react-router-dom';
```

Comment:
I dont see that you’re using BrowserRouter anywhere in this file. Is that import really necessary?